pkg/provider/k8s/k8s.go

```diff
@@ -24,10 +24,16 @@ import (
 
 	"github.com/prometheus/prombench/pkg/provider"
 
+	apiserverextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
lets use camel case everywhere to make it easyer to read:
`apiserverextensionsv1beta1 -> apiServerExtensionsV1Beta1` or something similar which is easy to read.
The same for the other places.